draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -628,13 +588,14 @@ Procedure:
 5.  domain = calculate_domain(PK, Q_1, (H_1, ..., H_L), header)
 6.  if domain is INVALID, return INVALID
 7.  D = P1 + Q_1 * domain + H_i1 * msg_i1 + ... + H_iR * msg_iR
-8.  C = Bbar * r2^ + Abar * r3^ + H_j1 * m^_j1 + ... + H_jU * m^_jU + D * c
-9.  cv = calculate_challenge(Abar, Bbar, C, (i1, ..., iR),
+8.  T =  Abar * r3^ + Bbar * r2^ + H_j1 * m^_j1 + ... +  H_jU * m^_jU
+9.  T = T + D * c
```

Comment:
Is this a line length issue? Also, switching the order of Abar and Bbar without switching r2 and r3 doesn't seem worth it to me :)

Reply:
Oh boy another breaking change 😅. Switched r2 with r3 in #276 (with updated fixtures)

Reply:
> Is this a line length issue?
Yeap sadly